deno publish Decorators are not valid here - Failed ensuring public API type output is valid

Version: 2.7.14 / JSR 0.14.3

```
Download https://jsr.io/@panva/jose/6.2.2/src/lib/ecdhes.ts
Download https://jsr.io/@panva/jose/6.2.2/src/lib/pbes2kw.ts
Download https://jsr.io/@panva/jose/6.2.2/src/lib/rsaes.ts
Download https://jsr.io/@panva/jose/6.2.2/src/lib/aesgcmkw.ts
Check mod.ts
Checking for slow types in the public API...
Check mod.ts
error: Failed ensuring public API type output is valid.

TS1206 [ERROR]: Decorators are not valid here.
  @Inject(JWT_MODULE_OPTIONS, {
  ^
    at file:///home/runner/work/denorid/denorid/packages/jwt/jwt_service.ts:29:3

You may have discovered a bug in Deno. Please open an issue at: https://github.com/denoland/deno/issues/
Command: /home/runner/.npm/_npx/91d31d47806d2506/node_modules/jsr/.download/v2.6.7/linux/deno publish
CWD: /home/runner/work/denorid/denorid/packages/jwt
Output: 
Child process exited with: 1
Error: Process completed with exit code 1.
```

I'm not sure why it doesn't allow me to decorate a `private readonly` property, but I think that's the problem here.

## Additional information
- I'm trying to publish my packages but the [CI](https://github.com/neonbyte1/denorid/actions/runs/25196248320/job/73877554878) fails with the message above
- The source code is [here](https://github.com/neonbyte1/denorid/blob/main/packages/jwt/jwt_service.ts#L29-L30)

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -93,7 +93,7 @@ deno_ast = { version = "=0.53.2", features = ["transpiling"] }
 deno_core_icudata = "0.77.0"
 deno_doc = "=0.199.0"
 deno_error = "=0.7.1"
-deno_graph = { version = "=0.108.2", default-features = false }
+deno_graph = { version = "=0.108.3", default-features = false }
 deno_lint = "=0.84.1"
 deno_media_type = { version = "=0.4.0", features = ["module_specifier"] }
 deno_native_certs = "0.3.0"

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -93,7 +93,7 @@ deno_ast = { version = "=0.53.2", features = ["transpiling"] }
 deno_core_icudata = "0.77.0"
 deno_doc = "=0.199.0"
 deno_error = "=0.7.1"
-deno_graph = { version = "=0.108.3", default-features = false }
+deno_graph = { version = "=0.108.4", default-features = false }
 deno_lint = "=0.84.1"
 deno_media_type = { version = "=0.4.0", features = ["module_specifier"] }
 deno_native_certs = "0.3.0"

--- a/cli/lsp/jsr.rs
+++ b/cli/lsp/jsr.rs
@@ -96,6 +96,7 @@ impl JsrCacheResolver {
           )]
           .into_iter()
           .collect(),
+          latest: Some(nv.version.clone()),
         })),
       );
       info_by_nv.insert(nv.clone(), Some(version_info));

